sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_access_control_client.py

```diff
@@ -0,0 +1,112 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from typing import TYPE_CHECKING
+
+from azure.core.tracing.decorator import distributed_trace
+
+from ._models import RoleAssignment, RoleDefinition
+from ._internal import KeyVaultClientBase
+
+if TYPE_CHECKING:
+    from typing import Any, Iterable
+
+
+class KeyVaultAccessControlClient(KeyVaultClientBase):
+    """Manages role-based access to Azure Key Vault.
+
+    :param str vault_url: URL of the vault the client will manage. This is also called the vault's "DNS Name".
+    :param credential: an object which can provide an access token for the vault, such as a credential from
+        :mod:`azure.identity`
+    """
+
+    # pylint:disable=protected-access
+
+    @distributed_trace
+    def create_role_assignment(self, role_scope, role_assignment_name, role_definition_id, principal_id, **kwargs):
+        # type: (str, str, str, str, **Any) -> RoleAssignment
+        """Create a role assignment.
+
+        :param str role_scope: scope the role assignment will apply over
+        :param role_assignment_name: a name for the role assignment. Must be a UUID.
+        :type role_assignment_name: str or uuid.UUID
+        :param str role_definition_id: ID of the role's definition
+        :param str principal_id: ID of the principal which will be assigned the role. This maps to the ID inside the
+            Active Directory. It can point to a user, service principal, or security group.
+        :rtype: RoleAssignment
```

Comment:
should be ~azure.keyvault.administration.RoleAssignment

Reply:
Sphinx links this correctly, the class is imported in this module.

Reply:
oh interesting, do you know for sure if microsoft docs will?

Reply:
I don't. Trial balloon! 🎈